frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -1013,7 +1016,10 @@ const HostDetailsPage = ({
               softwareUpdatedAt={host.software_updated_at}
               isSoftwareEnabled={featuresConfig?.enable_software_inventory}
               router={router}
-              queryParams={parseHostSoftwareQueryParams(location.query)}
+              queryParams={{
+                ...parseHostSoftwareQueryParams(location.query),
+                include_available_for_install: false,
+              }}
```

Comment:
This makes it so we add `include_available_for_install: false` on the host software page. But leave the my device > software page alone.

Trying to introduce this gnarly parameter in as few places as possible.